web/src/main/java/no/nav/foreldrepenger/fordel/web/app/rest/journalføring/FerdigstillJournalføringTjeneste.java

```diff
@@ -189,13 +198,26 @@ public void oppdaterJournalpostOgFerdigstillGenerellSak(String enhetId, ArkivJou
     }
 
 
-    public void oppdaterJournalpostMedTittelOgMangler(ArkivJournalpost journalpost, String nyJournalpostTittel, List<DokumenterMedNyTittel> dokumenterMedNyTittel, String aktørId, BehandlingTema behandlingTema) {
+    public void oppdaterJournalpostMedTittelOgMangler(ArkivJournalpost journalpost,
+                                                      String nyJournalpostTittel,
+                                                      List<DokumenterMedNyTittel> dokumenterMedNyTittel,
+                                                      String aktørId,
+                                                      BehandlingTema behandlingTema) {
         var journalpostId = journalpost.getJournalpostId();
         var kanal = journalpost.getKanal();
 
-        if ((nyJournalpostTittel != null || !dokumenterMedNyTittel.isEmpty()) && (MottakKanal.SELVBETJENING.getKode().equals(kanal) || MottakKanal.ALTINN.getKode().equals(kanal))) {
-            throw new FunksjonellException("FP-963071", String.format("Kan ikke endre tittel på journalpost med id %s som kommer fra Selvbetjening eller Altinn.", journalpostId),
+        if ((nyJournalpostTittel != null || !dokumenterMedNyTittel.isEmpty()) && List.of(MottakKanal.SELVBETJENING.getKode(),
+            MottakKanal.ALTINN.getKode()).contains(kanal)) {
+            // TFP-5791: Det er spesialhåndtering av tilfeller hvor klage/anke journalpost tittel ble endret i gosys uten at man har endret dokument tittel samtidig.
+            if (nyJournalpostTittel == null && dokumenterMedNyTittel.size() == 1) {
```

Comment:
Blir ikke dette for generell åpning - da tillater du vel endring av både IM og Søknad uten vedlegg?
Bør det legges på at unntaket ikke gjelder IM eller søknader (mens vedlegg og klage er ok)

Reply:
Noe usikker på hvordan det kan gjøres på en strukturert måte. 

GUI tillater ikke endring hvis IM eller søknad - det er kun systemet som gjør dokument tittel likt journalpost tittel om de er forskjellig (her var det en SBH som endret journalpost tittel i gosys men glemte å endre dokument tittel i samme sleng.

Så det er veldig spesielt for tilfeller hvor en oppgave i gosys blir opprettet (primært for klage og anke).

Reply:
Hvis det er validering frontend - så er det vel greit.